setup.cfg

```diff
@@ -25,6 +25,7 @@ install_requires =
     numpy >=1.9.3
     scipy
```

Comment:
I believe scipy is only needed in tests. Equally, we should add pandas in the "extra" section for tests.